src/Core/Framework/Bundle.php

```diff
@@ -159,7 +171,6 @@ private function registerEvents(ContainerBuilder $container): void
     {
         $definition = $container->getDefinition(BusinessEventRegistry::class);
         $definition->addMethodCall('addMultiple', [$this->getActionEvents()]);
-        $definition->addMethodCall('addClasses', [$this->getActionEventClasses()]);
```

Comment:
why did you remove that?

Reply:
I'm not sure why this was removed. I just added it again